src/pages/guides/ios.mdx

```diff
@@ -0,0 +1,220 @@
+
+Repository: https://github.com/Web3Auth/examples/raw/main/ios
+
+## Quick Start
+
+```bash
+git clone https://github.com/Web3Auth/examples.git
+cd ios
+# Open in Xcode
+```
+
+## How it works?
+
+---
+
+When integrating Web3Auth iOS SDK with Social Login the flow looks something like this:
+
+![Web3Auth Core - Social Login Flow](/images/implicit-flow.svg)
+
+- When a user logs in with `Google`, Google sends a JWT `id_token` to the app. This JWT token is sent to the Web3Auth SDK's login function.
+
+- Finally, on successful validation of the JWT token, Web3Auth SDK will generate a private key for the user, in a self custodial way, resulting in
+  easy onboarding for your user to the application.
+
+## Prerequisites
+
+---
+
+<Web3AuthPrerequisites />
+
+## Setup
+
+---
+
+### Setup your Web3Auth Dashboard
+
+<SetupWeb3AuthDashboard />
+
+- Add `(bundleId)://auth` in the **Whitelist URL** field of the Web3Auth Dashboard.
+
+## Using the Web3Auth SDK
+
+---
+
+### Installation
+
+#### Swift Package Manager
+
+If you are using the Swift Package Manager, open the following menu item in Xcode:
+
+`File > Swift Packages > Add Package Dependency`
+
+In the Choose Package Repository prompt add this url:
+
+```
+https://github.com/web3auth/web3auth-swift-sdk
+```
+
+#### Cocoapods
+
+```sh
+pod install Web3Auth
+```
+
+After Installation, the next step to use Web3Auth is to Initialize the SDK. However, the Initialization can be done in 2 ways,
+
+- [With `Web3Auth.plist`](#with-web3authplist)
+- [Without `Web3Auth.plist`](#without-web3authplist)
+
+Please note that these are the most critical steps where you need to pass on different parameters according to the preference of your project.
+Additionally, Whitelabeling and Custom Authentication have to be configured within this step, if you wish to customise your Web3Auth Instance.
+
+### Initialization
+
+#### With `Web3Auth.plist`
+
+In your application bundle add a plist file named `Web3Auth.plist` with the following information:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+  <key>ClientId</key>
+  <string>YOUR_OPENLOGIN_CLIENT_ID</string>
+  <key>Network</key>
+  <string>mainnet | testnet</string>
+</dict>
+</plist>
+```
+
+When you have a Web3Auth.plist and you have specified your Web3Auth configuration there, you can simply construct your Web3Auth instance with
+
+```swift
+import Web3Auth
+
+let web3auth = Web3Auth()
+```
+
+#### Without `Web3Auth.plist`
+
+If you don't have a Web3Auth.plist or want to use optional configurtion parameters, you can construct the Web3Auth instance with
+
+```swift
+import Web3Auth
+
+let web3auth = Web3Auth(W3AInitParams(
+  clientId: "<your clientId>",
+  network: .MAINNET,
+  sdkUrl: ...,
+  redirectUrl: ...,
+))
+```
+
+All the below code snippets are based on the above initialization.
+
```

Comment:
Add link to SDK Reference that for more information check the SDK reference for initialisation section